docs/reviewer-guide.md

```diff
@@ -23,6 +23,12 @@ Make a few passes over the code you want to review.
   index in a hash map on one of the lists and iterate over the other list. This
   trades `O(n^2)` with `O(n)`, which keeps kubevirt robust and scalable.
 
+## Release notes labels
+
+PRs have a ```release-notes``` section of their description and we use GitHub labels to filter release notes when we generate them at release time. For some PRs, kubevirt-bot is able to apply one of more of these labels but in general they need to be manually added.
```

Comment:
What happens if a PR matches multiple kinds? I don't think that's reflected in the current release notes section.

Reply:
Good point, I will add a para. The more labels the better imho (when applicable)
My thinking was that we'd filter them in terms of hierarchy of importance (as listed here but might need to be modified). So if a PR fixes a bug and has an kind/action-required label then it would appear only in the action-required section. 
I'm expecting a manual review prior to release, part of which will be to catch anything that gets auto-filtered into a wrong or less appropriate section.

Reply:
I added it to release.md rather than the reviewer-guide. Seemed logical to keep the more specific info in release.md and keep the reviewer guide lighter but link to release.md.